examples/real-world/webpack.config.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
   },
   plugins: [
     new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.HotModuleReplacementPlugin()
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
```

Comment:
Yea it would be undefined if you ran `npm start`, but not if you ran `NODE_ENV=production npm start`